llvm/test/CodeGen/AArch64/partial-reduce-sdot.ll

```diff
@@ -0,0 +1,69 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
```

Comment:
Is this test needed? test/Codegen/AArch64 should only contain code-gen related tests

Reply:
It seems like it's a hold over from a previous version. Removed now.